PendingReleaseNotes

```diff
@@ -83,6 +83,23 @@
   redistribution. In contrast, a stretch rule with a single-take configuration
   will not cause any data movement during the upgrade process.
 
+* CephFS: Command "ceph fs volume create" now allows users to pass metadata
+  and data pool names that should be used for creating the volume. In case
+  either or both of these pool names are not existing, the command will create
+  them on the go. Passing name of a non-empty pool will case the command to
+  abort as soon as it encounters it.
+
```

Comment:
Here's my suggestion for the whole paragraph:
```
CephFS: The "ceph fs volume create" command now allows users to pass metadata
and data pool names to be used for creating the volume.  These pools will be created if they
do not already exist.  Passing the name of an existing, non-empty pool will result in the command failing.

Reply:
I see why these three release notes felt so similar to you. I've kept them separate because all three are different features and have different tickets. Plus, the first release note is actually a different PR (see #61732), it's here only because it is a dependency.